sdk_lib/make_chroot.sh

```diff
@@ -47,6 +47,11 @@ DEFINE_string stage3_path "" \
   "Use the stage3 located on this path."
 DEFINE_string cache_dir "" "Directory to store caches within."
 
+DEFINE_string extra_portdir_overlay_path "" \
+  "Extra PORTDIR_OVERLAY direcory path outside of the chroot."
+DEFINE_string extra_portdir_overlay_name "" \
```

Comment:
Ah, I missed that detail. Would be good to resolve the path with something like `readlink -f` and explicitly test for that assumption, otherwise it will mysteriously break for someone else down the road.